config/client.lua

```diff
@@ -11,1557 +11,554 @@ return {
         zone = vec3(-29.53, -1103.67, 26.42), -- Where the finance menu is located
     },
 
-    vehicles = {
-        asbo = {
-            shop = 'pdm',
-        },
-        blista = {
-            shop = 'pdm',
-        },
-        brioso = {
-            shop = 'pdm',
-        },
-        club = {
-            shop = 'pdm',
-        },
-        dilettante = {
-            shop = 'pdm',
-        },
-        dilettante2 = {
-            shop = 'pdm',
-        },
-        kanjo = {
-            shop = 'pdm',
-        },
-        issi2 = {
-            shop = 'pdm',
-        },
-        issi3 = {
-            shop = 'pdm',
-        },
-        issi4 = {
-            shop = 'pdm',
-        },
-        issi5 = {
-            shop = 'pdm',
-        },
-        issi6 = {
-            shop = 'pdm',
-        },
-        panto = {
-            shop = 'pdm',
-        },
-        prairie = {
-            shop = 'pdm',
-        },
-        rhapsody = {
-            shop = 'pdm',
-        },
-        brioso2 = {
-            shop = 'pdm',
-        },
-        weevil = {
-            shop = 'pdm',
-        },
+    -- For the configuration below, it would first look for the vehicle in models. If not found, it would check for the category in categories. If the category is also not found, it would default to the default settings.
+    default = 'pdm',
+
+    categories = {
+        coupes = 'pdm',
+        boats = 'boats',
+        air = 'air',
+        -- super = {'pdm', 'luxury'},
+    },
+
+    models = {
```

Comment:
you dont need all of these here if the default is already pdm

Reply:
i let those for idiots that don't know how to add a item into table, but yeah i removed them